guidelines/index.html

```diff
@@ -2236,17 +2236,17 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
          </section>
          
          
-         <section class="informative" id="privacy-summary"><div class="header-wrapper"><h3 id="x5-6-privacy-considerations"><bdi class="secno">5.6 </bdi>Privacy Considerations</h3><a class="self-link" href="#privacy-summary" aria-label="Permalink for Section 5.6"></a></div><p><em>この節は規定ではない。</em></p>
+         <section class="informative" id="privacy-summary"><div class="header-wrapper"><h3 id="x5-6-privacy-considerations"><bdi class="secno">5.6 </bdi>プライバシーの考慮事項</h3><a class="self-link" href="#privacy-summary" aria-label="Permalink for Section 5.6"></a></div><p><em>この節は規定ではない。</em></p>
             
             
             
-            <p>Success Criteria within this specification which the Working Group has identified possible implications for privacy, either by providing protections for end users or which are important for web site providers to take in to consideration when implementing features designed to protect user privacy, are listed below. This list reflects the current understanding of the Working Group but other Success Criteria may have privacy implications that the Working Group is not aware of at the time of publishing.</p>
+            <p>この仕様内の達成基準のうち、エンドユーザに保護を提供するか、又は、利用者のプライバシーを保護する目的で設計された機能を実装する際にウェブサイト提供者が考慮することが重要であり、プライバシーに影響を与える可能性があるとワーキンググループが特定したものを次に示す。このリストはワーキンググループの現在の理解を反映したものであるが、他の達成基準にも、発行時点でワーキンググループが認識していないプライバシーに関する影響があるかもしれない。</p>
```

Comment:
ちょっと野暮ったいですが、次はいかがでしょう…？

```suggestion
            <p>この仕様内の達成基準のうち、エンドユーザに保護を提供するもの、又は、利用者のプライバシーを保護する目的で設計された機能を実装する際にウェブサイト提供者が考慮することが重要なものであって、プライバシーに影響を与える可能性があるとワーキンググループが特定したものを次に示す。このリストはワーキンググループの現在の理解を反映したものであるが、他の達成基準にも、発行時点でワーキンググループが認識していないプライバシーに関する影響があるかもしれない。</p>
```